pygtfs/loader.py

```diff
@@ -87,15 +87,15 @@ def append_feed(schedule, feed_filename, strip_fields=True,
                 read_records += 1
             except:
                 skipped_records += 1
-                print(f"Failure while writing {record}")
+                print("Failure while writing {}".format(record))
                 if not ignore_failures:
                     raise
             if i % chunk_size == 0 and i > 0:
                 schedule.session.flush()
                 sys.stdout.write('.')
                 sys.stdout.flush()
-        print(f'{read_records} records read for {gtfs_class}')
-        print(f'{skipped_records} records skipped for {gtfs_class}')
+        print('{0} records read for {1}'.format(read_records, gtfs_class))
```

Comment:
Maybe now that even debian dropped Python2, we should too. I see it hurt your pr with the f-strings. We should open a github issue to track dropping Python2.

Reply:
I agree- dropping Python 2 support should be a separate line of work.

Reply:
Do you mind changing this such that it will work with Python 2?